circuits/cpp/dockerfiles/Dockerfile.x86_64-linux-clang-tidy

```diff
@@ -2,22 +2,22 @@ FROM alpine:3.17 AS builder
 RUN apk update \
     && apk upgrade \
     && apk add --no-cache \
-        build-base \
-        clang15 \
-        clang15-extra-tools \
-        openmp-dev \
-        cmake \
-        ninja \
-        git \
-        curl \
-        perl \
-        bash \
-        python3
+    build-base \
+    clang15 \
+    clang15-extra-tools \
+    openmp-dev \
+    cmake \
+    ninja \
+    git \
+    curl \
+    perl \
+    bash \
+    python3
```

Comment:
Not important, but i actually think the old indentation was maybe clearer

Reply:
This just formatted on me. Not even sure how. I can revert